packages/multichain/src/scope/errors.test.ts

```diff
@@ -0,0 +1,33 @@
+import { Caip25Errors } from './errors';
+
+describe('Caip25Errors', () => {
+  it('requestedChainsNotSupportedError', () => {
+    expect(Caip25Errors.requestedChainsNotSupportedError().message).toBe(
```

Comment:
Nit: The error numbers aren't covered here. Maybe this is a good case for inline snapshots as well?

Reply:
added here: https://github.com/MetaMask/core/pull/4784/commits/86450dcb1d6127ffe05cf3678d6c73c10afd4b30